pkg/pillar/vcom/common.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2018-2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package vcom
+
+// BasePacket is the base packet for all other packets
+// it should be embedded in other packets.
+type BasePacket struct {
+	Channel uint `json:"channel"`
+}
+
+// TpmRequest is the request packet for TPM related requests
+type TpmRequest struct {
+	BasePacket
+	Request uint `json:"request"`
+}
+
+// ErrorResponse is the response packet for errors
+type ErrorResponse struct {
+	Error string `json:"error"`
+}
+
+// TpmResponseEk is the response packet for TPM Endorsement Key
+type TpmResponseEk struct {
+	Ek string `json:"ek"`
+}
+
+const (
+	baseChannelID = 1
+	// HostVPort is the port on which the host listens
+	HostVPort = 2000
+	// List of available channel, channel name should be in form of
```

Comment:
Yetus complains about these comments... Even if you decide not to follow my suggestions from the previous comment, it would be helpful to separate the `const` block into several ones with proper comments on each so that Yetus does not complain.